### PR TITLE
Updated qcbm tutorial.

### PR DESCRIPTION
**Pull request by mstechly**

### CODE REVIEW

File: intro/content/tutorial/qcbm.md [hunk 1]
```diff
@@ -488,7 +595,7 @@ resources:
 - name: qcbm
   type: git
   parameters:
-    url: "git@github.com:<your-github-username>/<your-git-repo-name>.git"
+    url: <your-github-username>/<your-git-repo-name>
```
Comment (simonwa7) [inline intro/content/tutorial/qcbm.md:598]: Why have you included this update? It doesn't seem right to me.

**Comment by simonwa7** [comment]
Have you tried rerunning the workflow and plotting the training?

File: intro/content/tutorial/qcbm.md (hunk 1 above)
Comment (mstechly) [reply intro/content/tutorial/qcbm.md]: Oh yes, my bad!
Just fixed it.

**Comment by mstechly** [comment]
@simonwa7 yes, it works well.

**Pull request merged**